docs/configuration/configuration.md

```diff
@@ -1181,6 +1181,10 @@ namespaces:
   [ - role: <string>
     [ label: <string> ]
     [ field: <string> ] ]]
+
+# Optional this determines which field of "Node.Status.Addresses" that discovery will get the address from first
+# default: NodeInternalIP,NodeInternalDNS,NodeExternalIP,NodeExternalDNS,NodeLegacyHostIP,NodeHostName
```

Comment:
Any reason why this uses a comma-separated string rather than an array?

Reply:
You are right, array would be better way.I originally wanted to keep the same form as `--kubelet-preferred-address-types`, but is is not necessary.